docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3-slim
+LABEL maintainer="OpenManageAnsible@Dell.com"
+
+ENV path /dellemc
+RUN mkdir $path
+WORKDIR $path
+
+RUN pip install "ansible<2.10" omsdk
+
+ADD https://github.com/dell/dellemc-openmanage-ansible-modules/archive/devel.tar.gz .
```

Comment:
Any reasons why are not using clone the branch and used install command? Is git package not supported in slim version?

Reply:
Any reason at all to fetch the code? I think the workflow will look like this: 1) clone this repo 2) build the image.

You see, everything's already available locally. I'm no container expert, but can't you build the modules from the cloned repository and copy them to the docker image?

Reply:
@rajeevarakkal Yes git is not installed on 'slim' version

@mariolenz These were the thoughts running on my mind while designing it:
* Customer need not clone the entire repository in order to use it, he can just copy this Dockerfile and build it on hist own. Also planning to push pre-built image to docker hub which also eliminates need to clone the repository from github.
* In future we are planning to integrate with Dockerhub, so that we can enable automated build generation with appropriate tagging in place.

With these in mind, I thought of writing standalone Dockerfile which is capable of generating image without any other dependencies.